test/test_generate_sarif.py

```diff
@@ -18,10 +18,10 @@
 
 class TestGenerateSarif:
     NO_CVE_SARIF_OUTPUT = json.loads(
-        """{"$schema": "https://json.schemastore.org/sarif-2.1.0.json", "version": "2.1.0", "runs": [{"tool": {"driver": {"name": "CVE Binary Tool", "semanticVersion": "3.2.2dev0", "rules": [{"id": "@cve-bin-tool/html_pdf_report", "name": "HTML & PDF Scan reports by CVE Binary Tool", "shortDescription": {"text": "HTML & PDF Scan reports by CVE Binary Tool"}, "fullDescription": {"text": "HTML & PDF reports provide detailed insight of the CVEs and make them easy to track. These results get automatically updated eachtime when the CVE Binary Tool GitHub Action runs. You can disable this just by dismissing the alert, then you won't receive any further HTML/PDF reports."}, "defaultConfiguration": {"level": "warning"}, "properties": {"tags": ["scan reports"], "precision": "medium"}}]}}, "results": []}]}"""
+        """{"$schema": "https://json.schemastore.org/sarif-2.1.0.json", "version": "2.1.0", "runs": [{"tool": {"driver": {"name": "CVE Binary Tool", "semanticVersion": "3.2.2dev0", "rules": [{"id": "@cve-bin-tool/html_pdf_sbom_report", "name": "HTML, PDF & SBOM Scan reports by CVE Binary Tool", "shortDescription": {"text": "HTML, PDF & SBOM Scan reports by CVE Binary Tool"}, "fullDescription": {"text": "HTML & PDF reports provide detailed insight of the CVEs and SBOM makes tracking dependency easy. These results get automatically updated eachtime when the CVE Binary Tool GitHub Action runs. You can disable this just by dismissing the alert, then you won't receive any further HTML/PDF/SBOM reports."}, "defaultConfiguration": {"level": "warning"}, "properties": {"tags": ["scan reports"], "precision": "medium"}}]}}, "results": []}]}"""
```

Comment:
Just noting that these will likely have to be regenerated if you add the extended text I suggested above.